contrib/poly2tri/poly2tri/sweep/sweep.cc

```diff
@@ -231,7 +231,7 @@ void Sweep::FillAdvancingFront(SweepContext& tcx, Node& n)
 
   while (node->next) {
     double angle = HoleAngle(*node);
-    if (angle > M_PI_2 || angle < -M_PI_2) break;
+    if (angle > PI || angle < -PI_2) break;

```

Comment:
Damn, this is embarassing... this kind of mistake in such a simple PR...

@mosra Thanks for noticing!
@kimkulling Sorry about this. Shall I create a PR for the fix or do you want to do the fix yourself?